capstone/src/test/java/com/google/sps/servlets/clubs/ClubServletTest.java

```diff
@@ -82,13 +83,14 @@ public void tearDown() {
   }
 
   @Test
-  public void doPost_registerNewValidClub() throws ServletException, IOException {
+  public void doPost_registerNewValidClub()
+      throws ServletException, IOException, GeneralSecurityException {
     when(request.getUserPrincipal()).thenReturn(principal);
     when(principal.getName()).thenReturn("test-email@gmail.com");
     when(request.getParameter(Constants.EXCLUSIVE_PROP)).thenReturn(null);
 
     doPost_helper();
-    clubServlet.doPostHelper(request, response, datastore, calendarService);
+    // clubServlet.doPostHelper(request, response, datastore, calendarService);
```

Comment:
nit: please remove commented out code if you don't need it anymore. Same for all the other instances in this file

Reply:
Done, thank you for catching this!